src/assets/styles/components/_buttons.scss

```diff
@@ -0,0 +1,46 @@
+.button {
+  border: 1.5px solid $border-grey;
```

Comment:
pixels cannot be broken in pieces. We can use just 1px or 2px (not 1.5px)
if you need to add 0.5px for border we can use `box-shadow`